### PR TITLE
Update pytest instructions in documentation

### PR DESCRIPTION
**Pull request by mcbarton**
This PR updates the pytest instructions in the documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ cd test
 ```
 and
 ```bash
-pytest -sv .
+pytest -sv test_xcpp_kernel.py
 ```
```
Comment (anutosh491) [inline README.md:56]: Ahh wait I am confused. So we create build and technically after running make install we have a test dir inside build. Do we also exoect the `test_xcpp_kernel.py` to be present there.

Comment (mcbarton) [reply README.md:56]: @anutosh491 It is on Linux and MacOS, as can be seen from the ci. Its not true for Windows (I found this out fixing the ci for Jupyter Kernel Test on Windows). I plan to fix this when I add instructions for building on Window

Comment (anutosh491) [reply README.md:56]: I am using a linux based device and I don't find this inside my build/test dir. will check again !

Comment (tharun571) [reply README.md:56]: There is no `test_xcpp_kernel.py` inside build/test. The way I think to check this test is to run this command inside the test dir with all the test files.

Comment (anutosh491) [reply README.md:56]: +1 Atleast the current cmake instruction don't move the `test_xcpp_kernel.py` file into `build/test` for me

**Comment by mcbarton** [comment]
@anutosh491 I've updated the instructions.Are you happy with them now?

**Review by anutosh491 — APPROVED** [review]
LGTM

**Pull request merged**